hollow/src/main/java/com/netflix/hollow/api/codegen/HollowAPIGenerator.java

```diff
@@ -170,6 +171,15 @@ public void setUseHollowPrimitiveTypes(boolean useHollowPrimitiveTypes) {
         this.useHollowPrimitiveTypes = useHollowPrimitiveTypes;
     }
 
+    /**
```

Comment:
This only impact Scalar type so might be a good idea to have the javadoc and code to reflect that ;)